tests/test_templates_embeddings.py

```diff
@@ -215,26 +220,68 @@ def circuit(x=None):
 
         with pytest.raises(ValueError) as excinfo:
             circuit(x=[np.sqrt(0.2), np.sqrt(0.8), 0, 0, 0])
-        assert excinfo.value.args[0] == "AmplitudeEmbedding requires the size of feature vector to be smaller than or equal to 2**len(wires), which is 4; got 5."
+        assert excinfo.value.args[0] == "AmplitudeEmbedding requires the size of feature vector to " \
```

Comment:
No biggie, but for future reference, you can also use:
```
with pytest.raises(ValueError, match="some string to match")
```
This allows you to just match a substring, rather than having to type the whole string out here in tests

Reply:
We had an issue with the raises() function before in a similar example...but yes, else that would be more elegant.